### PR TITLE
Add proprety in meta tags

### PR DESCRIPTION
**Pull request by SaraVieira**
Needed for the Open Graph protocol that's heavily used in SEO
https://ogp.me/

### CODE REVIEW

**Comment by SaraVieira** [comment]
Also can I have write access boss?

File: typed-html/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "axohtml"
-version = "0.1.0"
+version = "0.1.1"
```
Comment (ashleygwilliams) [inline typed-html/Cargo.toml:3]: let's do a release PR separately :)

**Review by ashleygwilliams — CHANGES_REQUESTED** [review]
nice. let's remove the version bump and do a release in a separate PR. also might be worth commenting the part of the standard this matches :) just for documentation sake

File: typed-html/Cargo.toml (hunk 1 above)
Comment (SaraVieira) [reply typed-html/Cargo.toml]: All done :)

File: typed-html/src/elements.rs [hunk 2]
```diff
@@ -78,6 +78,8 @@ declare_elements! {
         content: String,
         http_equiv: HTTPEquiv,
         name: Metadata,
+        // non standard, uses https://en.wikipedia.org/wiki/RDFa
```
Comment (ashleygwilliams) [inline typed-html/src/elements.rs:81]: <3

**Review by ashleygwilliams — APPROVED** [review]

**Pull request merged**